contracts/babylon/schema/babylon-contract.json

```diff
@@ -1,6 +1,6 @@
 {
   "contract_name": "babylon-contract",
-  "contract_version": "0.17.0",
+  "contract_version": "0.18.0",
```

Comment:
should contract version be `v1.0.0-rc.0`? comment also applies to other places

Reply:
yep just updated the schema